core/src/cs/eng1/piazzapanic/food/ingredients/Patty.java

```diff
@@ -18,10 +18,13 @@ public Patty(FoodTextureManager textureManager) {
   @Override
   public Texture getTexture() {
     String name = getType() + "_";
-    if (grilled) {
-      name += "grilled";
-    } else {
+
+    if (!useable) {
```

Comment:
Wouldn't it be better if we used one word to represent that the cooking failed, for consistency?
Over in `BasicChoppable` we use ruined instead.
To be honest it might be best to use consistent naming between grilling, cutting, etc.

Reply:
This is true, will change others to "ruined" then.